grub-core/loader/i386/slaunch.c

```diff
@@ -77,6 +77,8 @@ grub_slaunch_init_slrt_storage (int arch)
   /* Setup DCE and DLME information. */
   slr_dl_info_staging.hdr.tag = GRUB_SLR_ENTRY_DL_INFO;
   slr_dl_info_staging.hdr.size = sizeof(struct grub_slr_entry_dl_info);
+  slr_dl_info_staging.bl_context.bootloader = GRUB_SLR_BOOTLOADER_GRUB;
+  slr_dl_info_staging.bl_context.context = slparams.boot_params_addr;
   slr_dl_info_staging.dce_base = slparams.dce_base;
   slr_dl_info_staging.dce_size = slparams.dce_size;
   slr_dl_info_staging.dlme_entry = mle_header->entry_point;
```

Comment:
Base and size of DLME (aka. MLE) must also be passed to DCE, in addition to the entry point. In TXT, this information is passed through OS2SINIT, but there is no similar structure for AMD.

Reply:
https://github.com/TrenchBoot/grub/compare/ad13737578f935bcff302407860c2c6da7f73dab..a2af951d4f61ac7a5ea819eba49ce979c544a5ff#diff-33a2d35c8932ea248b7465a187fc4603651840c19cb98630f3d5276d9163d57bR85

Updated https://github.com/TrenchBoot/documentation/issues/23